src/ui/Card/Card.stories.jsx

```diff
@@ -455,6 +455,17 @@ function CarouselCardsContent({
   cards,
 }) {
   const slider = React.useRef(null);
+
+  useEffect(() => {
```

Comment:
Please check if you can recreate this implementation in volto.
Could not find any other way to refactor the dots except useEffect.

Reply:
The implementation is actually easy without the need for an useEffect see
the change I made directly in develop https://github.com/eea/volto-eea-design-system/commit/fe5eb46dc94abfeaaa2c79022e507b452b4bb36a